src/Jupyter/Magic/ConfigMagic.cs

```diff
@@ -25,7 +25,48 @@ public class ConfigMagic : AbstractMagic
         public ConfigMagic(IConfigurationSource configurationSource) : base(
             "config",
             new Documentation {
-                Summary = "Allows setting or querying configuration options."
+                Summary = "Allows setting or querying configuration options.",
+                Description = @"
+                    This magic command allows for setting or querying
+                    configuration options used to control the behavior of the
+                    IQ# kernel (e.g.: state visualization options), and to
+                    save those options to a JSON file in the current working
+                    directory.
+                ".TrimLeadingWhitespace(),
+                Examples = new []
+                {
+                    @"
+                        Print a list of all currently set configuration options:
+                        ```
+                        In []: %config
+                        Out[]: Configuration key                 Value
+                               --------------------------------- -----------
+                               dump.basisStateLabelingConvention ""BigEndian""
+                               dump.truncateSmallAmplitudes      true
+                        ```
+                    ",
```

Comment:
.TrimLeadingWhitespace()? (or .TrimStart()?)

Reply:
Good point, neither does quite the right thing here. I'm looking for something like Python's [`dedent()`](https://docs.python.org/3.8/library/textwrap.html#textwrap.dedent) function... will take another stab.